src/rad/wxfbmanager.cpp

```diff
@@ -26,6 +26,12 @@
 #include "model/objectbase.h"
 #include "rad/appdata.h"
 
+// Sometimes these macros get called with an empty argument which is not an error
+// but MSVC still produces a warning about too few arguments, disable it
+#ifdef _MSC_VER
+#pragma warning(disable:4003)
```

Comment:
You can also disable warnings in MSVC's project files, which I would prefer, as I don't like to add MSVC specific code in cpp files.

(I actually don't like to support MSVC at all, as it increases the maintenance burden and is proprietary)

Reply:
Since MSVC is like the "native" compiler on windows i am quite interested that it is supported. Plus compiling wxWidgets on my machine with MSVC running on one core is still faster than with gcc running on all four.